src/__stories__/success-feedback-screen-story.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import {SuccessFeedbackScreen} from '../feedback-screen';
+import {ButtonPrimary, ButtonSecondary} from '../button';
+
+export default {
+    title: 'Components/Screens/FeedbackScreen',
```

Comment:
You can't have two different story files with the same tile. This causes problems in storybook

Reply:
I did have problems while sepparating the story files until I gave a name to each story component correctly. In the end it seems to compose a correct index
```
- Screens
  - FeedbackScreen
        - ErrorFeedbackScreen
        - FeedbackScreen
        - InfoFeedbackScreen
        - SuccessFeedbackScreen
```
The preview is showing that index. Do you think there can be any other problem??

Reply:
In the past, defining using the same title in different story files caused issues (like stories disappearing while developing). If you don't get any problems, perhaps storybook already fixed this issue.

Reply:
I've seen some issues with hotreloading. Moreover, stories with no argTypes definition takes the definition that is set in other stories. So I'll follow your recomendation and give each story a different path. Thanks!

Anyway, I get some ugly urls with this last change:

`http://localhost:6006/?path=/story/components-screens-feedbackscreen-successfeedbackscreen--success`
I cannot get rid of that double "success" id.
- successFeedbackScreen comes from the url path: `title: 'Components/Screens/FeedbackScreen/SuccessFeedbackScreen'`
- the last `success` id comes from the name of the exported component in the story: `export const Success: StoryComponent<SuccessArgs> =`

This is my best trial... I cannot get storybook paths working correctly along with pretty urls...